src/legacy/core_plugins/data/public/search/aggs/buckets/terms.ts

```diff
@@ -58,17 +56,6 @@ export const termsAggFilter = [
   '!sum_bucket',
 ];
 
-const [orderAggSchema] = new Schemas([
-  {
-    group: AggGroupNames.None,
```

Comment:
this information is no longer present on agg configs, only aggFilter is kept all the rest is UI specific and should be handled by editor